.coveragerc

```diff
@@ -164,6 +164,7 @@ omit =
     homeassistant/components/envisalink/*
     homeassistant/components/ephember/climate.py
     homeassistant/components/epson/media_player.py
+    homeassistant/components/epsonworkforceprinter/sensor.py
```

Comment:
What do you think about naming it "epsonworkforce"? We usually use the product name but not the product category.